contents/blog/clickhouse-announcement.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2021-08-13
+title: We just made PostHog Open Source 1000x more scalable via ClickHouse
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+We're excited to announce a major under-the-hood upgrade for PostHog Open Source! Over the past few weeks, our team have been hard at work moving over from Postgres to ClickHouse, the same database that powers PostHog Cloud.
+
+This means that anyone can now get started using PostHog for free at scale - you'll no longer be limited to 10k monthly active users. Read on to find about more about the benefits and how we did this!
+
+## Why ClickHouse?
+
+ClickHouse is a database designed to enable fast, efficient analytics computations across multiple machines. We are moving [PostHog Open Source]() from PostgreSQL to ClickHouse because ClickHouse is a better overall experience on every level. Postgres runs out of capacity quickly and doesn’t support the advanced analytics we need for our product, and there is no way to scale PostgreSQL beyond a certain point.
+There are several benefits to using ClickHouse over PostgreSQL, such as:
+* Speed: ClickHouse is orders of magnitude faster than most other databases, mainly because of how tightly it couples storage and computing power, resulting in significantly quicker query response times.
+* Cost: With more efficient use of processing power, memory, and storage (plus horizontal scaling), you can use cheaper commodity boxes versus having to scale up using specialized (read: costlier) software.
+* Migrations: Moving over from another database? ClickHouse lets you create materialized versions of your data thanks to its advanced engine, resulting in exponentially faster migrations between deployments. In future, we'll also enable you to more easily migrate between PostHog Cloud and PostHog Self Hosted as both are backed by ClickHouse now.
```

Comment:
Minor quibble, but can we use 'enables you to' rather than 'lets you'? Just shifts the tone from one of permission to one of empowerment.